packages/orm/src/relations/ManyToManyCollection.ts

```diff
@@ -179,6 +179,19 @@ export class ManyToManyCollection<T extends Entity, U extends Entity>
     this.maybeApplyAddedAndRemovedBeforeLoaded();
   }
 
+  import(other: ManyToManyCollection<T, U>, findEntity: (e: U) => U): void {
+    function map(v: U[] | undefined): U[] | undefined {
+      if (v === undefined) return undefined;
+      const result = new Array<U>(v.length);
+      for (let i = 0; i < v.length; i++) result[i] = findEntity(v[i]);
+      return result;
+    }
+
+    this.#loaded = map(other.#loaded);
+    this.#addedBeforeLoaded = map(other.#addedBeforeLoaded);
+    this.#removedBeforeLoaded = map(other.#removedBeforeLoaded);
```

Comment:
These `import`s look just fine; they do make me a little nervous wrt to test coverage; i.e. do our `em.fork` tests probe relations for each each of the m2o/o2o/m2m/etc types to make sure they came over correctly? Can be a todo if not.

Reply:
all our tests pass in gql-service, so i'm sure it's fine :sweat_smile: